Rakefile

```diff
@@ -11,39 +11,19 @@ task :lint do
   sh "./gradlew clean lint"
 end
 
-# Usage:
```

Comment:
We decided to simplify the Rakefile for ease of maintenance since we usually always go for `rake unit_tests`.